app/services/trainees/filter.rb

```diff
@@ -29,6 +29,14 @@ def level(trainees, levels)
       trainees.where(query, *age_ranges.flatten)
     end
 
+    def record_source(trainees, record_source)
+      return trainees if record_source.blank? || record_source.size > 1
+
+      return trainees.with_manual_application if record_source.include? "manual"
```

Comment:
For what it's worth, i think we should be consistent with parenthesis vs non-parenthesis across the whole codebase but we can address this in a fix it Friday (or maybe it's just me being pedantic!)

Reply:
Yep, probably [Style/MethodCallWithArgsParentheses](https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/MethodCallWithArgsParentheses) could be leveraged here. [Adding a card for fix-it-friday](https://trello.com/c/vjj6Lnus/1755-agree-on-and-implement-method-parentheses)